### ISSUE TITLE
Add CONTRIBUTING.md

### ISSUE THREAD

**Issue by lambtho12** [EliverLara/candy-icons#70]
It would be nice to have a small CONTRIBUTING.md file in this repo with some style guidelines to help people that are willing to propose new icons.

This could include details of color palettes/gradients, line width, icon size, ...
Also determine properly which icons needs to be linked to the generic ones (e.g. text-editors for instance) and which ones can be standalone (e.g. Inkscape).

**Comment by Kopsi** [comment]
Yes. I would gladly contribute some icons if there were style guidelines.
